ReactWindows/ReactNative.Shared/Animated/PropsAnimatedNode.cs

```diff
@@ -37,7 +35,6 @@ public PropsAnimatedNode(int tag, JObject config, NativeAnimatedNodesManager man
             }
 
             _propMap = new JObject();
-            _diffMap = new ReactStylesDiffMap(_propMap);
```

Comment:
Was this one some kind of leftover/a future never fulfilled? I had noticed it was kind of empty.

Reply:
I think it was a useful abstraction for Android because they have ReadableMap and WriteableMap, and they may have needed to abstract over both. Here, we always use a "writable" JObject, so _byebye_